packages/roosterjs-editor-core/lib/coreApi/createPasteFragment.ts

```diff
@@ -113,10 +113,23 @@ export const createPasteFragment: CreatePasteFragment = (
                 .replace(/^ /g, NBSP_HTML)
                 .replace(/\r/g, '')
                 .replace(/ {2}/g, ' ' + NBSP_HTML);
-
-            const node = line == '' ? document.createElement('br') : document.createTextNode(line);
-
-            fragment.appendChild(index == 0 || index == lines.length - 1 ? node : wrap(node));
+            const textNode = document.createTextNode(line);
+
+            // There are 3 scenarios:
+            // 1. Single line: Paste as it is
+            // 2. Two lines: Add <br> between the lines
+            // 3. 3 or More lines, For first and last line, paste as it is. For middle lines, wrap with DIV, and add BR if it is empty line
```

Comment:
Just curious...If we ignore the # of lines, just always do "wrap with DIV, and add BR if it is empty line" for each line, would it still work?

Reply:
Consider you only copy 1 line, say "aaa", and the target content is "bbbccc", and you want to paste it between "bbb" and "ccc".
The expected paste result should be "bbbaaaccc".
But if we always wrap the content with DIV, when paste, it will become "bbb
aaa
ccc"

Reply:
I see.